azkaban-common/src/main/java/azkaban/jobExecutor/ProcessJob.java

```diff
@@ -46,12 +49,16 @@
 
   public static final String COMMAND = "command";
   public static final String AZKABAN_MEMORY_CHECK = "azkaban.memory.check";
-  public static final String NATIVE_LIB_FOLDER = "azkaban.native.lib";
```

Comment:
I would like to have all config keys in one place. But, I think refactoring any public static variable in these classes is a risk. Whether you want to keep things this way or refactor it and call deprecated is up to you. I'm fine with either.